.buildkite/scripts/e2e-pipeline/bootstrap.py

```diff
@@ -154,6 +186,7 @@ def run_elastic_stack(self) -> None:
         self.__download_elastic_package()
         self.__make_elastic_package_global()
         self.__clone_integrations_repo()
+        self.__scan_for_unsupported_processors()
```

Comment:
🤔 so if the elastic/integrations begin using an unsupported processor, our build breaks. 

I think we might end up needing a "known incompatibilities" list so that we can mark something as known and get our build back to green once we file an issue to address it.

Reply:
Yup this is the idea for the long term. As soon as we will have unsupported plugin (assuming the plugin really cannot support), we will have a white list to continue the E2E without failing. For now, for better visibility breaking CI (**_since it is important to know which integration will become problematic_**) seems better to me.

Reply:
I will comment this plan in the raising failure part of the source.